### PR TITLE
RST-1785 -  date input fields

### PR DESCRIPTION
**Pull request by zaparka**
Refactoring single date input field into a day/month/year fields.

### CODE REVIEW

File: app/models/forms/dob.rb [hunk 1]
```diff
@@ -1,10 +1,17 @@
 module Forms
   class Dob < Base
-    attribute :date_of_birth, Date
+    include ActiveModel::Validations::Callbacks
+
+    attr_reader :date_of_birth
+    attribute :day_date_of_birth, Integer
```
Comment (leoapost) [inline app/models/forms/dob.rb]: wouldn't be easier to just call these attributes: day, month, year?

Comment (zaparka) [reply app/models/forms/dob.rb]: @leoapost good point. let me try that.

File: config/locales/en.yml [hunk 2]
```diff
@@ -379,15 +390,15 @@ en:
       title: Number of children living with you
     dob:
       breadcrumb: Step 14 of 20
-      hint_html: Use this format <span class="dateFormatAccessible">D D / M M / Y Y Y Y</span>
+      hint_html: For example, 01 08 2007
```
Comment (leoapost) [inline config/locales/en.yml]: `DD MM YYYY` may be easier to understand that the day goes first and then the month.

Comment (zaparka) [reply config/locales/en.yml]: @leoapost that change is to be up to date with GDS pattern, not because I just decide to change it :)

**Pull request merged**